pages/CheckingPage.js

```diff
@@ -0,0 +1,66 @@
+import BasePage from '../pages/BasePage';
+
+class CheckingPage extends BasePage {
+
+    get standardCheckingButton (){return $('label[for="Standard Checking"]')}
+
+    get interestCheckingButton (){return $('label[for="Interest Checking"]')}
+  
+    get individualCheckingButton (){return $('#Individual')}
+  
+    get jointCheckingButton (){return $('#Joint')}
+  
+    get checkingNameInput (){return $('#name')}
+  
+    get checkingDepositInput (){return $('#openingBalance')}
+  
+    get newCheckingFormSubmitButton () {return $('#newCheckingSubmit')}
+
+    get newCheckingFormResetButton () {return $('#newCheckingReset')}
+
+    get newCheckingConfirmation () {return $('#new-account-conf-alert')}
+
+    get newCheckingErrorText () {return $('#new-account-error-msg')}
+
+    get switchChecking () { return $('.switch-label')}
+
+    get checkingBody () { return $('.even')}
+
+    get checkingFormNextButton () {return $('#transactionTable_next')}
+
+    get checkingFormPreviousButton () {return $('#transactionTable_previous')}
+
+    get checkingTableTextInfo(){return $('#transactionTable_info')}
+
+    get checkingSearchInput(){ return $('[type="search"]') }
+
+    async changeCheckingFormPageNext(){
+        await this.checkingFormNextButton.click()
+    }
+    async changeCheckingFormPagePrevious(){
```

Comment:
Same comment about making the name refer to the trasnactions table instead of the form